### PR TITLE
chore: add a test app for tooling manual testing

### PR DESCRIPTION
**Pull request by janechu**
# Description

This creates the scaffolding for in-browser manual testing of utilities in `@microsoft/fast-tooling`.

## Motivation & context

In order to test the Monaco Editor utilities that will be made, the Monaco Editor must be added to a test app to test and verify certain aspects of the editor for ease of use. This change begins to address that need by adding a setup for `webpack-dev-server`. At some later date this can be leveraged to do automated in-browser testing.

## Issue type checklist

- [x] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

**Adding or modifying component(s) in `@microsoft/fast-components` checklist**

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [x] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### CODE REVIEW

File: packages/tooling/fast-tooling/webpack.dev.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable no-undef */
+const merge = require("webpack-merge");
+const baseConfig = require("./webpack.common");
+
+module.exports = merge(baseConfig, {
+    devServer: {
+        contentBase: "./src/public",
+        open: true,
+        port: 7776,
```
Comment (awentzel) [inline packages/tooling/fast-tooling/webpack.dev.js:10]: Are we using the same port value as other local web servers?

File: packages/tooling/fast-tooling/webpack.common.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+                    loader: "worker-loader",
+                },
+            },
+        ],
+    },
+    plugins: [
+        new CleanWebpackPlugin(),
+        new MiniCssExtractPlugin(),
+        new HtmlWebpackPlugin({
+            title: "FAST",
```
Comment (awentzel) [inline packages/tooling/fast-tooling/webpack.common.js:69]: Could we add more specific title so it shows up in the browser tabs/hover etc.

Comment (janechu) [inline packages/tooling/fast-tooling/webpack.common.js:69]:
```suggestion
            title: "FAST tooling test application",
```

File: packages/tooling/fast-tooling/webpack.dev.js (hunk 1 above)
Comment (janechu) [reply packages/tooling/fast-tooling/webpack.dev.js:10]: Potentially, but in theory you wouldn't be running these at the same time.

**Pull request merged**